packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -1,6 +1,16 @@
 .dataviews-wrapper {
 	width: 100%;
+	min-height: calc(100% - 60px);
```

Comment:
The 60px is not part of the data views component, so it seems it's not something we should be applying to `.dataviews-wrapper` instead maybe it should apply to the current usage of DataViews in the `page-pages` component.

Reply:
Good call. Actually it seem I had this change before the `overflow` rule. It seems the `calc` is not needed any more.